packages/appstore/src/components/cfds-listing/cfds-listing.scss

```diff
@@ -620,6 +620,9 @@
     }
     &__new-password {
         margin-bottom: 8.6rem;
+        .dc-input__hint {
+            margin: 0 0 -5.3rem 1.3rem;
+        }
```

Comment:
I can see the same code in 3 different places, can you clarify on it, do you need to clean up the other places?

- cfds-listing
- cfd/cfd-dashboard
- trader/cfd-dashboard

Reply:
The css which is used by password modal is only cfd-listings.css.
cfd-dashboard is removed in the Removal PR WEBREL-1275'
trader/cfd-dashboard is also redundant.

Reply:
I think we can do the cleanup on the other places. Let me test again if something breaking by removing other occurences

Reply:
The trader/cfd-dashboard was kept as it was at the time of cfd removal from the trader package and its removal was missed after cfd worked fine as a separate package. The cfd-listing styling was basically the same css from the cfd package which was moved to appstore at the appstore implementation time. The main css file right now is the cfd-listing in appstore package.